src/saml2/response.py

```diff
@@ -435,7 +435,7 @@ def update(self, mold):
         self.response = mold.response
 
     def issuer(self):
-        return self.response.issuer.text.strip()
+        return self.response.issuer and self.response.issuer.text.strip()
```

Comment:
The First returns a string, the second a boolean. It would be Better to return a string

Reply:
Thank you for the feedback, I've updated the commit accordingly.